.storybook/main.js

```diff
@@ -5,6 +5,6 @@ module.exports = {
     '@storybook/addon-actions',
     '@storybook/addon-viewport',
     '@storybook/addon-storysource',
-    "storybook-addon-next-router",
+    '@storybook/addon-mock'
```

Comment:
How does this solve the issue? Please add screenshots of the fix as well.

Reply:
It was a mistake actually, because of some git issues, i will fix it asap